src/js/tech/tech.js

```diff
@@ -522,13 +523,25 @@ class Tech extends Component {
    *
    * @fires Tech#vttjsloaded
    * @fires Tech#vttjserror
-   * @fires Tech#texttrackchange
    */
   addWebVttScript_() {
     if (!window.WebVTT && this.el().parentNode !== null && this.el().parentNode !== undefined) {
+      const vtt = require('videojs-vtt.js');
```

Comment:
Yeah, we use the vtt.js option if we have it, if not we use the require if it is available (which it won't be for novtt builds), otherwise we use the cdn url.

The transform that you mention works like this:
[`aliasify`](https://github.com/benbria/aliasify) when passed [`false`](https://github.com/benbria/aliasify#stubbing-out-packages) for a package alias will turn the require call into an empty object `{}` before browserify includes in in the bundle.